scrapyd_k8s/api.py

```diff
@@ -3,8 +3,8 @@
 from .config import Config
 config = Config()
 from .logging_config import setup_logging
-logging_level = config.scrapyd().get('logging_level', 'INFO')
-setup_logging(logging_level)
+log_level = config.scrapyd().get('logging_level', 'INFO')
```

Comment:
uhm, and also the config option, primarily!

Reply:
What do you mean exactly? I don't understand the comment, sorry.